interfaces/sycl/DeviceCircularQueueBuffer.cpp

```diff
@@ -3,16 +3,26 @@
 #include "utils/logger.h"
 
 namespace device {
+
+#if defined(DEVICE_USE_GRAPH_CAPTURING) && defined(SYCL_EXT_INTEL_QUEUE_IMMEDIATE_COMMAND_LIST)
+auto sycl_queue_properties() {
+  return sycl::property_list(cl::sycl::property::queue::in_order{},
+                             cl::sycl::ext::intel::property::queue::no_immediate_command_list{});
+}
+#else
+auto sycl_queue_properties() { return sycl::property_list(cl::sycl::property::queue::in_order{}); }
```

Comment:
`cl::sycl::property_list`

(otherwise our CI currently complains, alas)

Reply:
Ok I changed it. Is cl::sycl required because the CI uses an old version of hipSYCL, @davschneller ? I think the latest adaptivecpp version also dropped the cl:: namespace as per SYCL 2020.

Reply:
Yup; the version in the CI is pretty dated at that point in time (still hipSYCL/ACPP 0.9.3).